src/modules/Auth/components/ResetPassword/ResetPassword.tsx

```diff
@@ -59,25 +60,38 @@ export default function ResetPassword() {
     handleSubmit,
   } = useForm<DataForget>({ mode: "onChange" });
 
-  const onSubmit = async (data: DataForget) => {     
+  const onSubmit = async (data: DataForget) => {
     try {
       const response = await axios.post(User_URls.resetPassword, data);
-      console.log(response);
       toast.success(response.data.message);
       navigate("/login");
     } catch (error: any) {
       toast.error(error.response.data.message || "An error occurred");
```

Comment:
@Ahmed-Serag19 
I think you did make a function that handles the errors before by checking for the backend errors patterns first and then the axios error message and then a generic error message. We can utlize this method and include in it the toast message as well if we wish.
This will make our error handling more centeralized.
it will be just something like that `handleError(error)`